.github/copilot-instructions.md

```
@@ -0,0 +1,1772 @@
+- **Type Checking**: TypeScript v5+
+- **Commit Linting**: commitlint with @commitlint/config-conventional
+- **Git Hooks**: Husky v9+ with lint-staged
+- **CI/CD**: GitHub Actions
+- **Deployment**: Netlify/Cloudflare Pages with NuxtHub
+- **Monitoring**: Sentry for error tracking
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)